### ISSUE TITLE
Indenting problems, unnecessary font tags inserted into message divs?

### ISSUE THREAD

**Issue by jwoglom** [thelounge/thelounge#2167]
![image](https://user-images.githubusercontent.com/192620/37103511-6aaa286e-21f8-11e8-8475-38c0bdadf75c.png)

This seems to be occurring randomly, not just when followed by a status message, with multiple users. It looks to me as though it is caused by additional insertions of  `<font style="vertical-align: inherit">`, as shown below.

Offending message:
```
<div class="msg message" id="msg-4047" data-time="2018-03-07T18:34:09.175Z" data-from="XXX">
	<span class="time tooltipped tooltipped-e" aria-label="7 March 2018, 13:34:09"><font style="vertical-align: inherit;"><font style="vertical-align: inherit;">
		13:34
	 </font></font></span>
	<font style="vertical-align: inherit;"><span class="from"><span role="button" class="user color-2" data-name="XXX"><font style="vertical-align: inherit;">+ XXX </font></span></span><span class="content"><span class="text"><font style="vertical-align: inherit;">[message]</font></span></span></font><span class="from">
			<span role="button" class="user color-2" data-name="XXX"><font style="vertical-align: inherit;"></font></span>
	</span>
	<span class="content">
		<span class="text"><font style="vertical-align: inherit;"></font></span>

	</span>
</div>
```

Message where this doesn't occur:
```
<div class="msg message" id="msg-4054" data-time="2018-03-07T18:34:52.371Z" data-from="XXX">
	<span class="time tooltipped tooltipped-e" aria-label="7 March 2018, 13:34:52">
		13:34
	</span>
	<span class="from">
			<span role="button" class="user color-6" data-name="XXX">+XXX</span>
	</span>
	<span class="content">
		<span class="text">[message]</span>

	</span>
</div>
```

**Comment by xPaw** [comment]
Nowhere does Lounge use `<font>` tag, are you sure that's not some of your extensions doing that?

**Comment by jwoglom** [comment]
I think you're right. I found this reddit post: https://www.reddit.com/r/reactjs/comments/7hbtqx/am_i_missing_somthing/dqrcvsh/

It looks like Chrome was somehow convinced that the page needed translation, and added the `<font style="vertical-align: inherit;">` on its own. I just tried right-clicking and pressing "Translate to English" and the issue immediately popped up; so the issue seems to be with the integrated Google Translate functionality in Chrome.
